promql/promqltest/testdata/functions.test

```diff
@@ -759,32 +752,32 @@ eval instant at 1m sum_over_time(metric9[1m])/count_over_time(metric9[1m])
   {} -Inf
 
 eval instant at 1m avg_over_time(metric10[1m])
-  {} 0
+  {} 9.98846567431158e+307
```

Comment:
The point here is that two large numbers average to 0, so here again, we should have a longer range, like `2m` or `1m1s`.

Reply:
Get it. I changed evaluation range to `2m`.